.travis.yml

```diff
@@ -40,9 +40,7 @@ deploy:
     skip_cleanup: true
     api_key: $GITHUB_TOKEN
     name: ${PACKAGE_VERSION}
-    file:
-      - "./dist/cmd-resume.js"
-      - "./dist/cmd-resume.min.js"
+    file: "./dist/cmd-resume*.js"
```

Comment:
tested this?

Reply:
Can you remove this? i'd rather not touch the release stuff unless the linter told you to do this?

Reply:
The linter had an error with file because its linter only supports a single line for file. I'll revise it

Reply:
Weird, it works as all releases have those files. Maybe just `./dist/*.js` and pull down a docker image and see if it packages both files?

Reply:
Reverted the file.
Travis gives 2 warnings that aren't blocking
<img width="483" alt="screenshot 2018-11-08 13 31 36" src="https://user-images.githubusercontent.com/30458968/48198922-bf3fee80-e35a-11e8-9e57-044d9b8d9461.png">